flexget/plugins/sites/newpct.py

```diff
@@ -62,31 +62,24 @@ def parse_download_page(self, url):
         torrent_id = None
         if 'newpct1.com' in url:
             url_format = NEWPCT1_TORRENT_FORMAT
-            torrent_id_prog = re.compile('function openTorrent.*\n.*\{.*(\n.*)+window\.location\.href =\s*\".*\/(\d+.+)\";')
-            torrent_ids = soup.findAll(text=torrent_id_prog)
-            log.debug('searching openTorrent script')
-            if torrent_ids:
-                match = torrent_id_prog.search(torrent_ids[0])
-                if match:
-                    torrent_id = match.group(2)
         else:
             url_format = NEWPCT_TORRENT_FORMAT
 
-            torrent_id_prog = re.compile("(?:parametros\s*=\s*\n?)\s*{\s*\n(?:\s*'\w+'\s*:.*\n)+\s*'(?:torrentID|id)"
+        torrent_id_prog = re.compile("(?:parametros\s*=\s*\n?)\s*{\s*\n(?:\s*'\w+'\s*:.*\n)+\s*'(?:torrentID|id)"
                                          "'\s*:\s*'(\d+)'")
+        torrent_ids = soup.findAll(text=torrent_id_prog)
+        if len(torrent_ids):
+            match = torrent_id_prog.search(torrent_ids[0])
+            if match:
+                torrent_id = match.group(1)
+        if not torrent_id:
+            torrent_id_prog = re.compile('function openTorrent.*\n.*\{.*(\n.*)+window\.location\.href =\s*\".*\/(\d+).*\";')
             torrent_ids = soup.findAll(text=torrent_id_prog)
-            if len(torrent_ids):
+            log.debug('torrent ID not found, searching openTorrent script')
```

Comment:
This doesn't really make sense if `torrent_ids` is empty/none.

Reply:
There are two attempts to find torrent id, first regexp with 'parametros', then regexp with opentorrent. The log is a reference to the first regexp failure. Should I remove the log?